app/views/subscriptions/_subscription_fields.html.erb

```diff
@@ -13,7 +13,11 @@
   </div>
   <div class="row">
     <p class="form-description">Enter a Concept ID for a single collection. (Example: C1234567-PODAAC)</p>
-    <%= text_field_tag('subscription[CollectionConceptId]', @subscription['CollectionConceptId'], class: 'required full-width') %>
+    <% if edit %>
```

Comment:
according to the [text_field_tag](https://apidock.com/rails/v5.2.3/ActionView/Helpers/FormTagHelper/text_field_tag) and [select_tag](https://apidock.com/rails/ActionView/Helpers/FormTagHelper/select_tag) docs, you can use true or false for the disabled option. So I think you can set the disabled fields without the if/else blocks.
So you could have `class: "required full-width #{'disabled' if edit}", disabled: edit`.
if you're going to use it this way, maybe `edit` should be a bit more descriptive, i.e. `edit_form` maybe. On the groups form we used `new_form`, that's a possibility too.